portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,71 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+/*NAVBAR*/
+
+.navbar {
+  overflow: hidden;
+  background-color: #333;
+  top: 10;
+  /*potion: fixed;*/
+  z-index: 2;
+}
+
+.navbar a {
+  float:left;
+  display: block;
+  color: #f2f2f2;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 16px;
+}
+
+/*makes games go under the c button and not all the way to left*/
+.cdropdown {
```

Comment:
The recommented style is to leave one space after the /* and before the */

Also not clear what "games" is referring to here.

Reply:
done